test/hosts-config.ts

```diff
@@ -1,3 +1,3 @@
 // These must match the hosts configured in .travis.yml
-export const testHost = "testhost";
-export const corsHost = "corshost";
+export const testHost = "localhost";
+export const corsHost = "localhost";
```

Comment:
Unfortunately this is necessary to cause IE to treat the different addresses (different ports) as different origins so that CORS behaviour can be tested. IE treats `localhost:1234` and `localhost:5678` as the same origin 😕

Relevant test failure: https://travis-ci.org/improbable-eng/grpc-web/builds/265692040#L1210